helpers/constants.py

```diff
@@ -185,12 +185,12 @@ def add_role(self, name):
 }
 
 DISABLED_VAULTS = [
-    "0xb6bd5ae3d5f78a6bb04bbb031e24fa9c2bbd090d",
-    "0x85e1cacae9a63429394d68db59e14af74143c61c",
-    "0xaf9ab64f568149361ab670372b16661f4380e80b",
-    "0xfdb9e5a186fb7655ac9cd7caff3d6d4c6064cc50",
-    "0x0c2153e8ae4db8233c61717cdc4c75630e952561",
-    "0x53c8e199eb2cb7c01543c137078a038937a68e40"
+    "0xB6bd5ae3d5F78A6Bb04bBb031E24fA9C2BbD090d",
```

Comment:
everything should be checksum now? why was it lower before?

Reply:
Before hand we sometimes used lower case and sometimes used checksum which causes some confusion. I would prefer to keep everything checksummed from now on